### PR TITLE
Add empty natural language gem packages

### PR DESCRIPTION
**Pull request by blowmage**

### CODE REVIEW

**Comment by blowmage** [comment]
@quartzmo Thoughts?

**Comment by quartzmo** [comment]
It seems a little premature to add these now, but my understanding is that we are doing it to unblock @jmuk so that he can update or renew #809 for both language and speech. In this case I'm all for it since we should definitely prioritize his help with the Gapic client code!

I'm just looking it over for typos, then I'll merge. Thanks @blowmage!

File: google-cloud-speech/lib/google/cloud/speech.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+require "google-cloud-speech"
+require "google/cloud/speech/project"
+
+module Google
+  module Cloud
+    ##
+    # # Google Cloud Language
+    #
+    module Language
```
Comment (quartzmo) [inline google-cloud-speech/lib/google/cloud/speech.rb]: Critical c/p typo: change to `module Speech`.

**Comment by quartzmo** [comment]
@blowmage I only found one typo. Other than that, ready to merge.

**Comment by quartzmo** [comment]
I have signed the CLA and am okay with these commits being contributed to Google.

**Pull request merged**